README.md

```diff
@@ -13,7 +13,7 @@ If compared to saving websites with `wget -mpk`, `monolith` embeds all assets as
 ### Installation
     $ git clone https://github.com/Y2Z/monolith.git
     $ cd monolith
-    $ cargo install
+    $ cargo install --path .
```

Comment:
I'm probably less familiar with cargo than most programmers who use Rust, but wouldn't `--path .` be redundant, since we're running that command in the crate's directory anyway?

Reply:
cargo install works without problems but with a warning message in 1.39.0-nightly version
warning: Using `cargo install` to install the binaries for the package in current working directory is deprecated, use `cargo install --path .` instead